showcase.json

```diff
@@ -2746,5 +2746,27 @@
     "twitter": "",
     "youtube": "",
     "submittableId": ""
-  }
+  },
+{
+  "name": "IC based storage service",
+  "description": "IC based storage service to store and organize any kind of files. Yes, at first glance it might be similar to AWS S3 or Google drive where user can store files, share the access, have a direct http link to the file, organize the files within the directories etc. But it works on the IC platform.",
```

Comment:
Could you please provide a more comprehensive and informative description?

Potentially update this to:

"IC-based storage service to store and organize any kind of files that works with platform deployed on the IC"

Reply:
ICS2 project helps to store any kind of files in the IC platform. User can sore a file, organize files with directories, each resource has a direct link to open a file or download it. If file is under the directory (or even subdirectory) then user can fetch the same file by direct link like /r/{hash} or name-based approach /i/{directory_name}/{file_name}. 
ICS2 supports simple UI interface that exposed by /i/ path even it is a backend canister.

We can think about ICS2 as a simple version of the aws s3 :).

So, the vision "IC-based storage service to store and organize any kind of files that works with platform deployed on the IC" might be good as well.

Reply:
The new description is more clear! Thank you for the update!